crates/turbopack-ecmascript/src/transform/mod.rs

```diff
@@ -34,6 +34,8 @@ pub enum EcmascriptInputTransform {
     /// well as any imports that are only used by those exports.
     ///
     /// It also provides diagnostics for improper use of `getServerSideProps`.
+    NextJsPageSsr,
+    // When enabled, this applies all transforms needed for Next.js to all Ecmascript
     NextJs,
```

Comment:
> all transforms needed for Next.js

That sounds a bit weird since it does not apply e. g. `NextJsPageSsr`.

Maybe just rename it to `NextJsFont` and change the comment

Reply:
That's what it was originally 😅 . This way we avoid individual options for each transform when they'd likely all need to be applied at once anyway.

Instead, what do you think about clarifying the comment here that at the moment only the font transform is applied?